mesheryctl/internal/cli/root/system/start.go

```diff
@@ -221,13 +221,20 @@ func start() error {
 
 			temp, ok := utils.Services[v]
 			if !ok {
-				return errors.New("unable to extract component version")
+				return errors.New(fmt.Sprintf("no such service `%s` exists.", v))
```

Comment:
@zakisk if this is user facing, it'd be helpful to expand on this message.
```suggestion
				return errors.New(fmt.Sprintf("No Docker Compose service exists for Meshery component `%s`.", v))
```